openmls/src/treesync/diff.rs

```diff
@@ -295,16 +286,19 @@ impl<'a> TreeSyncDiff<'a> {
         ciphersuite: Ciphersuite,
         group_id: GroupId,
         credential_bundle: &CredentialBundle,
+        own_leaf_index: LeafNodeIndex,
```

Comment:
Similar to the module file, shall we tro to get rid of all the "own"s in here and have it general?
There are of course requirements on the input and available keys. But that's really independent of the "own" now.

Reply:
Yes, I'll rename those input variables and change the docs to reflect the requirements.